libs/island-ui/core/src/lib/Tag/Tag.tsx

```diff
@@ -37,7 +37,7 @@ export const Tag = forwardRef<HTMLButtonElement & HTMLAnchorElement, TagProps>(
       onClick,
       variant = 'blue',
       active = false,
-      disabled,
+      disabled = false,
```

Comment:
won't this be `false` by default? same with the other props that have `false` default value